README.md

```diff
@@ -0,0 +1,19 @@
+# archipelag 
+
+##What is it?
+
+An application for NGOs to extend events scope using social media.
+
+First commits application was developed on hackathon HACKSILESIA 2017.
```

Comment:
I'm not very good at english, but I think
"Application's first commits was" is better.
the "'s" at the end of application word indicated that the "first commits" are from the Application.
If you have doubts, plase ask someone who is better, but I think the line above is wrong (but I also can be wrong (: ).